pr2eus/robot-interface.l

```diff
@@ -44,6 +44,60 @@
   (:time-to-finish ()
     (ros::ros-debug "[~A] time-to-finish ~A" ros::name-space time-to-finish)
     time-to-finish)
+  (:action-result-cb (msg)
+    "
+    display error messages from [depending on robots]/follow_joint_trajectory/result
+    "
+    ;; The following error messages will not appear for the first (send *ri* :angle-vector)
+    ;; To display it, use :wait-interpolation or (ros::spin-once groupname)
+    (let ((status-num (send (send msg :status) :status))
+          (status-word '(PENDING
```

Comment:
@tkmtnt7000 can we use https://github.com/jsk-ros-pkg/jsk_roseus/blob/4d45a0db6022ac960d8b17f7fa1447c8c72fcc95/roseus/euslisp/actionlib-commstate.l#L3 ?

Reply:
Thank you for suggestion.
Yes, we can use it. I updated in [6df961c](https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/460/commits/6df961cd10fddfebd9bfdd8d71b849c14e9e25bd)